src/API/sensors/sensor.controller.js

```diff
@@ -0,0 +1,42 @@
+import Sensor from './sensor.model';
+
+function get(req, res, next) {
+    Sensor.get(req.params.id).then((sensor) => {
+        res.json(sensor);
+        return next();
+    }).error((e) => next(e));
+}
+
+function create(req, res, next) {
+    var instance = new Sensor(req.body);
+
+    instance.saveAsync()
+        .then((sensor) => res.json(sensor))
+        .error((e) => next(e));
+}
+
+function update(req, res, next) {
+    Sensor.findByIdAndUpdate(req.params.id, req.body, {
+        new: true
+    })
+        .exec()
+        .then((sensor) => res.json(sensor))
+        .catch((e) => next(e));
```

Comment:
we have mongoose native promises and bluebird promises (we pomisify mongoose in index.js)
bluebird promises have an [error](http://bluebirdjs.com/docs/api/error.html) method, but mongoose promises dont.
mongoose can use 3rd party promises, but it didn`t work for me - for example, mongoose stopped handling [CastError](http://mongoosejs.com/docs/api.html#index_Mongoose-CastError) - it`s thrown, when we a using an objectId, which does not meet the [format](https://docs.mongodb.org/manual/reference/method/ObjectId/)